benchmarks/gups/gups.cpp

```diff
@@ -143,14 +165,13 @@ int main(int argc, char* argv[]) {
   printf("Kokkos GUPS Benchmark\n");
   printf(HLINE);
 
-  srand48(1010101);
-
   Kokkos::initialize(argc, argv);
 
-  int64_t indices = 8192;
-  int64_t data    = 33554432;
-  int64_t repeats = 10;
-  bool useAtomics = false;
+  int64_t indices       = 8192;
+  int64_t data          = 33554432;
+  int64_t repeats       = 10;
+  bool useAtomics       = false;
+  AccessPattern pattern = AccessPattern::random;
```

Comment:
should we either make the default atomics == false pattern = permute, or atomics = true and pattern = random?

That would avoid race conditions in the default mode?

Reply:
The RandomAccess benchmark in the HPC Challenge tolerates some amount of errors, so it might not be surprising to people if there are race conditions in our default GUPS execution. I'm happy to change it though if anyone has a strong opinion.